Source/OxyPlot/Series/BarSeries/LinearBarSeries.cs

```diff
@@ -158,11 +158,8 @@ public override void Render(IRenderContext rc)
             this.VerifyAxes();
 
             var clippingRect = this.GetClippingRect();
-            rc.SetClip(clippingRect);
```

Comment:
Why is this one not like the others? (i.e. it defers to the `Clipped` method, rather than setting the clipping rectangle here or in `RenderBars` just once). Is this just missing the `using var _ = rc.AutoResetClip(clippingRect);` statement?

(Note: if the push/pop methods check for duplicates, then even with the clipped methods repeating the work, the performance should be better already).

Reply:
I didn't want to go into too much detail optimising the clipping calls in this PR. There are probably many places in the code where we could do so. In fact, I think ideally series shouldn't have to be concerned with clipping at all, clipping could be set once for a given combination of X and Y axis and then all series can just do their stuff without having to consider clipping. But I think I'd like to leave this for a future PR.

Anyway I'll change the two places you mentioned as they are really obvious and quickly optimised.